clang/test/Driver/aarch64-implied-sve-features.c

```diff
@@ -0,0 +1,78 @@
+// RUN: %clang -target aarch64-linux-gnu -march=armv8-a+sve %s -### 2>&1 | FileCheck %s --check-prefix=SVE-ONLY
```

Comment:
Does this test need `REQUIRES: aarch64-registered-target`?

Reply:
Proposed change makes sense, and I'll add it to this PR, but bear in mind, I'm pulling in here an mainline llvm-project commit.